src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/tests/Sidebar.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
```

Comment:
We changed to try to use flow in tests as well now. That already have indicated some bugs, we have overseen without flow.